Source/Scene/GltfLoader.js

```diff
@@ -1504,6 +1578,116 @@ function loadStructuralMetadata(
   return structuralMetadataLoader;
 }
 
+function loadAnimationSampler(loader, gltf, gltfSampler) {
+  const animationSampler = new AnimationSampler();
+
+  const inputAccessorId = gltfSampler.input;
+  animationSampler.input = loadAccessor(loader, gltf, inputAccessorId);
+
+  const gltfInterpolation = gltfSampler.interpolation;
+  switch (gltfInterpolation) {
+    case "STEP":
+      animationSampler.interpolation = InterpolationType.STEP;
+      break;
+    case "CUBICSPLINE":
+      animationSampler.interpolation = InterpolationType.CUBICSPLINE;
+      break;
+    case "LINEAR":
+    default:
+      animationSampler.interpolation = InterpolationType.LINEAR;
+      break;
+  }
```

Comment:
since the enum names are an exact match, you can use:

```
animationSampler.interpolation = defaultVaue(
  InterpolationType[gltfInterpolation], 
  InterpolationType.LINEAR
);
```

I've seen other examples in the codebase like this, such as these lines from `MetadataEnum`:

```
const valueType = defaultValue(
  MetadataComponentType[enumDefinition.valueType],
  MetadataComponentType.UINT16
);  
```